common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst

```diff
@@ -2848,7 +2848,7 @@ set to 0).
    <tr style="color: #c0c0c0">
    <td><strong>param1</strong></td>
    <td>Type (0=as 1=gs)</td>
-   <td>Speed type (0=Airspeed, 1=Ground Speed). Set to 0 before v2.50, otherwise ignored.</td>
+   <td>Speed type . Set to 0 before v2.50, otherwise ignored.</td>
```

Comment:
we probably don't need to mention v2.50 anymore.

Reply:
fixed